Tasks/Operation/01_ManageEvents.md

```diff
@@ -0,0 +1,116 @@
+[!!User interface Events](../UserInterface/03a_List.md)
+[!!Process Orchestration](../../ActindoWorkFlow/Overview/01_General.md)
+
+# Manage the events
+
+An event is an occurrence that originates a task. For example, when inconsistencies in stock are found, the *Differences in stock* event creates a task to check the stock inconsistencies manually.
+
+You can create events manually in the *Tasks* module. However, most events are automatically created by the system when installing a plugin or module, such as the *Actindo Workflow Process Execution Failed*. 
+
+When the *Process Orchestration* (also known as *Workflows*) module is installed, the *Actindo Workflow Process Execution Failed* event is automatically created in the *Tasks* module. This event creates a task as soon as an error occurs in a process action. The workflow process in the *Process Orchestration* module is then paused until the specified task is performed manually by the responsible user. For detailed information about the workflow processes, see the [Process Orchestration](../../ActindoWorkFlow/Overview/01_General.md) documentation. 
+
+You can create an event manually, edit an existing event, and delete an event that is no longer needed.
+
+
+
+## Create an event
+
+You can create an event to define a scenario where a task must be created.
```

Comment:
Vorschlag:
You can create an event to define a scenario that requires a user to perform a task.